spec/rails_helper.rb

```diff
@@ -34,10 +34,22 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
+  config.before(:suite) do
```

Comment:
Seems like, database cleaner works incorrectly because in your feature specs user cannot be created